cmd/kubeadm/app/cmd/init.go

```diff
@@ -323,6 +323,17 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
+	// ReadSentinelFile check sentinelfile exist
+	if data, err := kubeadmutil.ReadSentinelFile(); err != nil {
+		// WriteSentinelFile is record kubeadmin init to sentinelfile
+		if err := kubeadmutil.WriteSentinelFile("kubeadm init"); err != nil {
+			glog.Errorf("[init] cannot write sentinel file: %v", err)
+		}
+		if data != "" {
```

Comment:
this won't be reached if `err == nil` from here:
`if data, err := kubeadmutil.ReadSentinelFile(); err != nil `